packages/core/src/commands/index.ts

```diff
@@ -450,4 +451,9 @@ export default class CommandsModule extends Module<CommandsConfig & { pStylePref
     this.commands = {};
     this.active = {};
   }
+
+  // Dummy property to ensure ComponentDragEventProps type is included in the build
+  dummyComponentDragEventProps() {
+    return undefined as ComponentDragEventProps | undefined;
+  }
```

Comment:
@artf this is the 'cleanest' way I found to get `ComponentDragEventProps` into the build types. Only importing/exporting doesn't work due the tree-shaking, but wondering if you've got another idea to get that type in the build without modifying the logic

Reply:
Fixed on 84ef292, exporting the type in `src/index` did the trick